registrations/management/commands/delete_signups_with_expired_payments.py

```diff
@@ -0,0 +1,37 @@
+from datetime import timedelta
+
+from django.core.management import BaseCommand
+from django.utils.timezone import localtime
+
+from registrations.models import SignUpPayment
+
+
+class Command(BaseCommand):
+    help = (
+        "Delete signups and signup groups that have payments that have expired at least the given "
+        "amount of days ago. The payments will be deleted along with the signups."
+    )
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--threshold_days",
+            type=int,
+            default=14,
+            help="How many days ago the payments should have expired before they are deleted."
+            "Default = 14.",
+        )
+
+    def handle(self, *args, **options):
+        threshold_datetime = localtime() - timedelta(days=options["threshold_days"])
```

Comment:
This may wobble +- 1 hour due to DST transition? Maybe not a problem

Reply:
Shouldn't be a problem as these payments will have been considered "deleted" for two weeks with the default threshold days option.